### PR TITLE
Add discord link and links trusted section logos

### PR DESCRIPTION
**Pull request by bdougie**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update
- [ ] Release

## Description
instead of the Signup button I am adding a link to the Discord on the homepage and nav. I am also adding a link in the README

## Related Tickets & Documents
N/A

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
Screenshot 2020-05-30 06 45 59

## Added tests?

- [ ] yes
- [x] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [x] readme
- [ ] no documentation needed

### CODE REVIEW

**Comment by yg** [comment]
@bdougie What is we add links to **logos** in the trusted section?

**Comment by bdougie** [comment]
> @bdougie What is we add links to logos in the trusted section?

Not following? What do you mean by links?

**Comment by yg** [comment]
> Not following? What do you mean by links?

I mean URLs for GraphQL, NextJS, etc

**Comment by bdougie** [comment]
Ah yes, that is a good idea.

File: src/components/Hero.js [hunk 1]
```diff
@@ -48,34 +48,16 @@ function Hero({handleLogIn}) {
               <small>Trusted By</small>
             </li>
             <li>
-              <img
-                alt="graphql"
-                src={graphql}
-              />
+              <img alt="graphql" src={graphql} />
```
Comment (yg) [inline src/components/Hero.js:51]: @bdougie This is I'm talking about, Link to GraphQL website!

File: README.md [hunk 2]
```diff
@@ -74,6 +74,9 @@ This project uses GitHub as a database. When you login, you will be presented wh
 
 This project uses the sw-precache to kickstart an offline cache. The offline cache only registers in production. If service needs to be manually removed make an **unregister** call from the registerServiceWorker.js import. 
 
+## 🍕Community
```
Comment (yg) [inline README.md:77]: Added Space, Markdown Standard for better readability, and reduce text congestion!

```suggestion
## 🍕Community

```

File: src/components/Hero.js (hunk 1 above)
Comment (yg) [reply src/components/Hero.js:51]: Screenshot

@bdougie for this!

File: README.md [hunk 3]
```diff
@@ -74,6 +74,10 @@ This project uses GitHub as a database. When you login, you will be presented wh
...
```
Comment (yg) [inline README.md:77]:
```suggestion
## 🍕 Community
```

@bdougie text space too! I forgot to suggest this in old suggestion!

**Comment by yg** [comment]
@bdougie Can we make a separate PR for this or continue here itself?

https://github.com/open-sauced/open-sauced/pull/583#pullrequestreview-421404135

**Comment by bdougie** [comment]
I think it is fine to continue. This PR has not increased scope too much. If it was a larger change, I would move it to another PR, but in this case, we went from adding discord links to adding discord and friends links--not a big deal.

**Pull request merged**